src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Hierarchical.hpp

```diff
@@ -432,10 +432,14 @@ class Hierarchical<VolumeDim, tmpl::list<Tags...>> {
 
   /*!
    * \brief The \f$\alpha_i\f$ values in the Krivodonova algorithm.
+   *
+   * The default size is the number of allowed Legendre polynomials
+   * (`Spectral::maximum_number_of_points<Spectral::Basis::Legendre`)
    */
   struct Alphas {
     using type = std::vector<double>;
-    static constexpr OptionString help = {"The alpha coefficients"};
+    static constexpr OptionString help = {
+        "The alpha parameters of Krivodonova limiter"};
```

Comment:
How about `The alpha parameters by Krivodonova`? (nitpicking here.. trying to get rid of redundancy in option descriptions)

Reply:
I don't like that phrasing, personally. I've updated to `The alpha parameters of the Krivodonova limiter`